packages/shared/src/components/FeedItemComponent.tsx

```diff
@@ -138,6 +141,13 @@ export default function FeedItemComponent({
 
   switch (item.type) {
     case 'post': {
+      if (
+        !!item.post.pinnedAt &&
+        squad?.currentMember?.flags?.collapsePinnedPosts
```

Comment:
Technically the items source should have this knowledge I think.

Reply:
sorry, I missed this piece of feedback -- .the flags info is indeed also available via `item.post.source.currentMember.flags...`, but then if I use this, calling the mutation does not update the UI, since we are updating the squad flags cache on mutation success: https://github.com/dailydotdev/apps/blob/ce183c6fe9855d1795509af1a16660b97d55f7ee/packages/shared/src/hooks/squads/useSquadActions.ts#L70

We could get around this by invalidating the feed query cache, but then we are making an additional API call which we don't really need: https://github.com/dailydotdev/apps/blob/bfb276714e1aa6eeefe3a262bd092457c461baf7/packages/shared/src/hooks/squads/useSquadActions.ts#L56-L72